NewRelicBrowser.php

```diff
@@ -1,392 +1,456 @@
 <?php
-
 /**
+ * New Relic Browser Plugin developed by rtCamp
+ *
+ * @package     NewRelicBrowser
+ * @author      rtCamp
+ * @license     GPL-2.0+
+ *
  * Plugin Name: New Relic Browser
- * Plugin URI: http://www.rtcamp.com
+ * Plugin URI:  http://www.rtcamp.com
  * Description: New Relic Browser Monitoring plugin.
- * Version: 1.0.5
- * Author: rtCamp
- * Author URI: http://www.rtcamp.com
- * License:  MIT
+ * Version:     1.0.5
+ * Author:      rtCamp
+ * Author URI:  http://www.rtcamp.com
+ * License:     MIT
  * License URI: http://opensource.org/licenses/mit-license.html
+ * Text Domain: rt-new-relic
  */
 
-if ( ! defined( 'RTP_NEW_RELIC_API_KEY' ) ){
+if ( ! defined( 'RTP_NEW_RELIC_API_KEY' ) ) {
 	define( 'RTP_NEW_RELIC_API_KEY', '6155aee398970036405f017b9f788801ed32f23e208f2d4' );
 }
-if ( ! defined( 'RTP_NEW_RELIC_API_URL' ) ){
+if ( ! defined( 'RTP_NEW_RELIC_API_URL' ) ) {
 	define( 'RTP_NEW_RELIC_API_URL', 'https://rpm.newrelic.com/api/v2/partners/857/accounts' );
 }
-if ( ! defined( 'RTP_NEW_RELIC_BROWSER_URL' ) ){
+if ( ! defined( 'RTP_NEW_RELIC_BROWSER_URL' ) ) {
 	define( 'RTP_NEW_RELIC_BROWSER_URL', 'https://api.newrelic.com/v2/browser_applications.json' );
 }
+if ( ! defined( 'RTP_RESOURCE_VERSION' ) ) {
+	define( 'RTP_RESOURCE_VERSION', filemtime( __FILE__ ) );
+}
```

Comment:
Remove this constant. You should get modified time of css/js files and use it for versioning while enqueuing.

Reply:
i solved this issue. please review changes.